utils/http.go

```diff
@@ -325,6 +349,17 @@ func (h *HTTP) EnsureCert(defaultPath string) (err error) {
 
 	log.Warningf("Generating self signed key and cert to %v %v.", h.KeyFile, h.CertFile)
 
+	certDir := path.Dir(defaultCertPath)
+	if _, err = os.Stat(certDir); os.IsNotExist(err) {
+		log.Warningf("Self-signed TLS certs directory %v doesn't exist, creating it now", certDir)
+
+		err := os.MkdirAll(certDir, 0644)
```

Comment:
Nevertheless, if you think it's worth to have this `mkdir -p`, lets add it. I just wonder who ever needs these self signed certs, cloud service like Slack or Pagerduty refuse to send webhooks to endpoints with self signed certs.

Reply:
This code basically does `mkdir -p` ;)